sep490-frontend/src/app/modules/shared/directives/form-field-error.directive.ts

```
@@ -0,0 +1,67 @@
+    if (this.control) {
+      translate.onLangChange
+        .pipe(takeUntil(this.destroy$))
+        .subscribe((): void => this.showErrors(this.errors || {}));
+    }
+  }
+
+  ngOnInit(): void {
+    if (this.control) {
+      this.control.errors$
+        .pipe(
+          tap(errors => (this.errors = errors)),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)